components/core/src/clp_s/log_converter/log_converter.cpp

```
@@ -18,6 +20,8 @@ using clp_s::log_converter::CommandLineArguments;
 using clp_s::log_converter::LogConverter;
 
 namespace {
+constexpr size_t cMaxRetries{3};
+constexpr std::chrono::seconds cInitialBackoff{1};
```

Cppcheck (2.20.0)

[style] The function 'get_filename' is never used.

(unusedFunction)